### ISSUE TITLE
mcAfee Endpoint security detected a threat numpy 1.22.3

### ISSUE THREAD

**Issue by SallyGuan** [numpy/numpy#21234]
### Steps to reproduce:

Python:3.10.3
Numpy:1.22.3
VS code:

**About...**
System Information  
Computer Name: N-20KEPC0YB1DH 
Device name	N-20KEPC0YB1DH
Processor	Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz   1.90 GHz
Installed RAM	8.00 GB (7.84 GB usable)
Device ID	A91413EC-A322-403A-9C2A-B5484309A59B
Product ID	00330-80000-00000-AA955
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Edition	Windows 10 Enterprise
Version	20H2
Installed on	‎10/‎25/‎2021
OS build	19042.1526
Experience	Windows Feature Experience Pack 120.2212.4170.0

   
McAfee Data Exchange Layer  
Version number: 6.0.3.551 
Build date: Thursday, December 23, 2021 
DXL Connected State: Connected 
Broker DNS Name: DXLInt02 
Broker Address: 135.251.47.98 
Broker Port Number: 8883 
   
   
McAfee Agent  
Version number: 5.7.5.504 
Status: Managed 
SuperAgent: Peer to Peer 
Last security update check: 3/23/2022 11:54:51 AM 
Last agent-to-server communication: 3/23/2022 1:08:07 PM 
Agent to Server Communication Interval (every): 2 hours 
Policy Enforcement Interval (every): 30 minutes 
Agent GUID: {faa48ca4-b1d3-11e9-1f6e-4aa4726a45eb} 
ePO Server/Agent Handler  
DNS Name: chzppmmfeahh01.nsn-intra.net 
IP Address: 135.251.187.27 
Port Number: 443 
DNS Name: cshjvpmsecepo01.ad4.ad.alcatel.com 
IP Address: 172.24.208.16 
Port Number: 443 
DNS Name: CSHJVPMSECAH01 
IP Address: 117.144.188.135 
Port Number: 443 
   
   
McAfee DLP Endpoint  
Version number: 11.6.400.342 
Language: English (United States) 
   
   
McAfee Endpoint Security  
Version number: 10.7 
For module and content information, view the About page   
in the McAfee Endpoint Security Client   
  
Copyright (C) 2022 McAfee LLC.  
All Rights Reserved.  
www.mcafee.com  
    

### Error message:

```shell
similar with https://github.com/numpy/numpy/issues/19040
Have you had the similar report for the version of Numpy 1.22.3?
would you please contact McAfree about this?
```

**Comment by SallyGuan** [comment]
![2022-03-23 13_23_32](https://user-images.githubusercontent.com/33344443/159632858-9a3b018a-9603-417d-bdcb-271986509fef.png)

**Comment by SallyGuan** [comment]
C:\Users\q19242\OneDrive - Nokia\0 - Common\0 - python learn\greenshot>pip.exe install numpy --proxy 10.x.x.10:80 --trusted-host pypi.org
Collecting numpy
  Using cached numpy-1.22.3-cp310-cp310-win_amd64.whl (14.7 MB)
Installing collected packages: numpy
Successfully installed numpy-1.22.3

**Comment by mattip** [comment]
[here](https://kc.mcafee.com/corporate/index?page=content&id=KB85567) is a link to their services with more information

**Comment by SallyGuan** [comment]
Ok. I will email McAfree about it.

**Comment by bashtage** [comment]
Probably the same issue in #21230 . It may be the binary in there is being blocked by AV software.

**Comment by seberg** [comment]
I am going to close this hoping that they fixed it.  If we need to put some weight behind any such request for fixing the anti-virus software, comment or open a new issue.
